pkg/datarepair/repairer/repairer.go

```diff
@@ -36,7 +36,7 @@ var (
 type Config struct {
 	MaxRepair    int           `help:"maximum segments that can be repaired concurrently" releaseDefault:"5" devDefault:"1"`
 	Interval     time.Duration `help:"how frequently repairer should try and repair more data" releaseDefault:"1h" devDefault:"0h5m0s"`
-	Timeout      time.Duration `help:"time limit for uploading repaired pieces to new storage nodes" default:"10m0s"`
+	Timeout      time.Duration `help:"time limit for uploading repaired pieces to new storage nodes" devDefault:"10m0s" releaseDefault:"2h"`
```

Comment:
So `2h` to upload single piece?

Reply:
The problem is that repair uploads are running into some strange error message and we want to make sure it has nothing to do with the timeout.